modules/tensor_mechanics/src/materials/ADComputeMultipleInelasticStress.C

```diff
@@ -127,7 +127,7 @@ ADComputeMultipleInelasticStress::initialSetup()
 
   if (isParamValid("damage_model") && !_damage_model)
     paramError("damage_model",
-               "Damage Model " + _damage_model->name() +
```

Comment:
You mean _damage_model could be nullptr here?

Reply:
Oh no what am I saying...

Reply:
You can say `damage_model = some_material` in the input file and `some_material` cannot be cast to `DamageBase`.

Reply:
For example, change https://github.com/idaholab/moose/blob/2270e3210b1f7151cf7dde712331bbdf64baf868/modules/tensor_mechanics/test/tests/scalar_material_damage/ad_scalar_material_damage.i#L97
to `damage_model = damage_index`, and since damage_index cannot be cast to DamageBase, it should go down this branch.

Reply:
thanks @hugary1995 for the inspiration!